torch/nn/modules/linear.py

```diff
@@ -53,11 +51,11 @@ def __init__(self, in_features, out_features, bias=True):
         self.reset_parameters()
 
     def reset_parameters(self):
-        init.kaiming_uniform_(self.weight, a=math.sqrt(5))
+        std = 1 / self.in_features ** 0.5
+        bound = std * 3 ** 0.5
+        init.uniform_(self.weight, -bound, bound)
```

Comment:
Can you use instead `kaiming_uniform_` with a potential different `a`? This is very similar to what we had before, the difference is that there is a `math.sqrt(3)` factor difference, see https://github.com/pytorch/pytorch/commit/f9a99d5504818e2d8211dd9229665fb732a9c61e#diff-531f4c06f42260d699f43dabdf741b6dR55

Reply:
```suggestion
        init.kaiming_uniform_(self.weight, nonlinearity='linear')
```
would be the more intuitive way to use `kaiming_uniform` (rather than with the leaky relu nonlinearity).